src/probabilistic/CMakeLists.txt

```diff
@@ -1,22 +1,20 @@
-
-include(ZeekSubdir)
-
-include_directories(BEFORE
-                    ${CMAKE_CURRENT_SOURCE_DIR}
-                    ${CMAKE_CURRENT_BINARY_DIR}
+zeek_add_subdir_library(
+    probabilistic
+    SOURCES
+        BitVector.cc
+        BloomFilter.cc
+        CardinalityCounter.cc
+        CounterVector.cc
+        Hasher.cc
+        Topk.cc
 )
 
-set(probabilistic_SRCS
-    BitVector.cc
-    BloomFilter.cc
-    CardinalityCounter.cc
-    CounterVector.cc
-    Hasher.cc
-    Topk.cc)
-
-bif_target(bloom-filter.bif)
-bif_target(cardinality-counter.bif)
-bif_target(top-k.bif)
-bro_add_subdir_library(probabilistic ${probabilistic_SRCS})
-
-add_dependencies(bro_probabilistic generate_outputs)
+# Treat BIFs as builtin (alternative mode).
```

Comment:
This is 7 lines to do 3 lines worth of work 😃

Reply:
We could also add a new 'keyword' to `zeek_add_subdir_library` for doing that. Maybe `ALT_BIFS` or something like that. That would make the call side less verbose. :)

Reply:
That would be better. I need to look back at what "alternative mode" is too, as it feels like there has to be a better name for that.

Reply:
Naming is hard. 😉

Reply:
Ah, it's a thing from `bifcl`. Unfortunately, there isn't any documentation for the arguments taken by that. Bifcl.cmake refers to a `-a` argument, but then uses `-s` later. Looking at the bifcl source, there's a `-p` argument as well. Internally, both set a variable named `alternative_mode` to true. I'm going to guess we could rename it to something like "static mode", which is more descriptive about what it actually does.